### PR TITLE
add(apps & websites): opengraph and clinicnearme

### PR DESCRIPTION
**Pull request by swsalim**
I hope it's okay to submit 2 of my websites on the same PR, if not, I'll remove one and create another PR. Thank you.

| Name                 | Link                 |
| -------------------- | -------------------- |
| Opengraph | https://www.opengraph.xyz/ |
| Clinicnearme | https://www.clinicnearme.sg/ |

OpenGraph is a tool to preview, check and generate Open Graph meta tags for your website.

Clinicnearme is a tool to search for clinics, general practitioners, hospitals, and more in Singapore

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] I have read and followed the [contribution guidelines](.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -306,6 +306,8 @@
 - [Sitesauce](https://sitesauce.app/) - A static version of your website in one click.
 - [Hello Sun](https://hellosun.brussels/) - Helping Brussels go solar.
 - [Nuxt](https://nuxtjs.org/) - Build your next Vue.js application with confidence with NuxtJS.
+- [OpenGraph](https://www.opengraph.xyz/) - Preview Open Graph Meta Tags for Your Website.
+- [ClinicNearMe.SG](https://www.clinicnearme.sg/) - Find 24-Hour Clinics, GPs, Hospitals & More in Singapore.
```
Comment (innocenzi) [inline README.md:310]: These descriptions do not respect the conventions of the list. Do not use title case.

**Review by innocenzi — COMMENTED** [review]
Submitting two items at once for the list is fine, but we've recently cleaned up the Apps & Websites section because people were submitting a lot of websites that were not necessarily awesome. For this reason, we've decided to go very strict about what we accept there. 

See: 
- https://github.com/aniftyco/awesome-tailwindcss/pull/288
- https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md#apps--websites

I think your sites look good (I especially like the OpenGraph preview, which I suggest you PR to [TinyHelpers](https://github.com/stefanjudis/tiny-helpers)), but I don't think they are suited for this list.

**Comment by joshmanders** [comment]
I agree with @innocenzi. We want to be a truly awesome list of stuff, not "here's everything ever made in Tailwind"

**Comment by swsalim** [comment]
> Submitting two items at once for the list is fine, but we've recently cleaned up the Apps & Websites section because people were submitting a lot of websites that were not necessarily awesome. For this reason, we've decided to go very strict about what we accept there.
> 
> See:
> 
> * #288
> * https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md#apps--websites
> 
> I think your sites look good (I especially like the OpenGraph preview, which I suggest you PR to [TinyHelpers](https://github.com/stefanjudis/tiny-helpers)), but I don't think they are suited for this list.

@innocenzi Thanks for sharing the links, wasn't aware of the clean up you guys did in the past.

Just wanted to be sure that these 2 websites are not suited for this list or do you want me to fix the description following the convention?

Thanks for the suggestion for OpenGraph :)

**Comment by innocenzi** [comment]
> Just wanted to be sure that these 2 websites are not suited for this list or do you want me to fix the description following the convention?

They are not suited for the list. The comment for the description was just because you checked the box in the PR about respecting the name/description conventions, but it was not actually respected. :)

**Comment by swsalim** [comment]
ah okay, that explained it. Thanks.

**Pull request closed without merging**